integration-tests/starboard_integration_test.go

```diff
@@ -0,0 +1,70 @@
+package integration_tests
+
+import (
+	"context"
+	"os/exec"
+
+	apiextentions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	starboardCmd = "./../bin/starboard"
+)
+
+var _ = Describe("Starboard CLI", func() {
+
+	BeforeEach(func() {
+		// setup
+	})
+
+	Describe("Running version command", func() {
+		It("should print the current version of the executable binary", func() {
+			cmd := exec.Command(starboardCmd, []string{"version"}...)
+			output, err := cmd.Output()
+			Expect(err).ShouldNot(HaveOccurred())
+			Expect(string(output)).To(Equal("Starboard Version: {Version:dev Commit:none Date:unknown}\n"))
+		})
+
+	})
+
+	Describe("Running init command", func() {
+		It("should initialize Starboard", func() {
+			cmd := exec.Command(starboardCmd, []string{"init", "-v", "3"}...)
```

Comment:
Check if we can use http://onsi.github.io/gomega/#gexec-testing-external-processes